Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -341,6 +342,101 @@ int EmbeddedFrag::findNeighbor(
   return -1;
 }
 
+bool EmbeddedFrag::matchToTemplate(const RDKit::INT_VECT &ringSystemAtoms, unsigned int ring_count) {
+  CoordinateTemplates& coordinate_templates = CoordinateTemplates::getRingSystemTemplates();
+  if (!coordinate_templates.hasTemplateOfSize(ringSystemAtoms.size())) {
```

Comment:
It's probably worth a comment somewhere to discuss why we only use templates that 100% overlap with the ring system. Like, why not use a template that matches _some_ of the right system?